src/wepy/resampling/novelty/novelty.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+class Novelty():
+
+    cycle_counter = 0
+    clone_merge_iter = 0
+    sample_counter = 0
```

Comment:
Not relevant since these will be removed, but these should be in ALL_CAPS and have docstrings.

Reply:
They have been removed. Thanks for the information I will correctly document them next time.